src/components/App/App.js

```diff
@@ -1,17 +1,46 @@
 import React from 'react';
+import './App.css';
 
 const App = props => {
+	props.loadStoreList();
```

Comment:
This should probably be in a lifecycle method (componentDidMount?)

Otherwise this will be called whenever App rerenders causing us to fetch the list multiple times and having side effects in render which is something we try and avoid